pkg/handlers/replicas.go

```diff
@@ -102,6 +103,7 @@ func MakeReplicaReader(defaultNamespace string, clientset *kubernetes.Clientset)
 			lookupNamespace = namespace
 		}
 
+		s := time.Now()
```

Comment:
This might be more valuable as an actual prometheus metric. Having it in the logs is nice, but it means a person is going to make judgments on a few values as opposed to being able to collect and aggregate real data into graphs etc.

Reply:
I agree. I think it is in there already 👍 This is another example of a log message we should probably delete.